spec/models/spam_report_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+
+describe SpamReport do
+  let(:first_spam) { create(:work, spam: true, posted: true) }
+  let(:second_spam) { create(:work, spam: true, posted: true, authors: first_spam.authors) }
+  let(:third_spam) { create(:work, spam: true, posted: true, authors: second_spam.authors) }
+
+  before(:example) do
+    ArchiveConfig.SPAM_THRESHOLD = 10
```

Comment:
You can stub it: `allow(ArchiveConfig).to receive(:SPAM_THRESHOLD).and_return(10)`.